src/features/search/components/SearchDropdown.tsx

```
@@ -110,80 +144,206 @@ const SearchDropdown: React.FC<SearchDropdownProps> = ({
-            <div className="px-3 py-2 text-xs font-medium text-gray-500 uppercase tracking-wider">
+            <div className="px-3 py-2 text-xs font-medium text-gray-500 uppercase tracking-wider bg-gray-50 border-b border-gray-100">
               Recent Searches
             </div>
             {recentSearches.slice(0, 3).map((search, index) => (
               <div
                 key={`recent-${index}`}
-                className="cursor-pointer px-4 py-3 hover:bg-gray-50 transition-colors duration-150 flex items-center gap-2"
-                onClick={() => onSelectItem(search)}
+                className="cursor-pointer px-3 py-3 hover:bg-gray-50 transition-colors duration-150 flex items-center gap-2"
+                onClick={() => onSelectItem(search, -1)}
               >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)